src/Components/Facility/ConsultationForm.tsx

```diff
@@ -1365,7 +1365,7 @@ export const ConsultationForm = ({ facilityId, patientId, id }: Props) => {
                         className="col-span-6"
                         ref={fieldRef["procedure"]}
                       >
-                        <FieldLabel>Procedures</FieldLabel>
+                        <FieldLabel>Procedure Suggestions</FieldLabel>
```

Comment:
Let's keep the text in i18n locale files

Reply:
Only in Consultation.json file inside the en directory ? Since there aren't any other files that contain texts related to this.

Reply:
yup